interview/inventory/urls.py

```diff
@@ -12,5 +16,6 @@
     path('languages/', InventoryLanguageListCreateView.as_view(), name='inventory-languages-list'),
     path('tags/', InventoryTagListCreateView.as_view(), name='inventory-tags-list'),
     path('types/', InventoryTypeListCreateView.as_view(), name='inventory-types-list'),
+    path('day/', InventoryPostDayListCreateView.as_view(), name='inventory-day-list'),
```

Comment:
Is this a good naming convention for the url?

Reply:
Changes made.